packages/cmf/src/componentState.js

```diff
@@ -13,7 +14,7 @@ export function getStateAccessors(dispatch, name, id, DEFAULT_STATE) {
 			);
 		},
 		initState(initialState) {
-			const state = DEFAULT_STATE.merge(initialState);
+			const state = (DEFAULT_STATE || new Map()).merge(initialState);
```

Comment:
i think in getStateAccessors parameters, DEFAULT_STATE = new Map() is a better fit